app/js/services/page-provider.js

```diff
@@ -0,0 +1,31 @@
+angular.module('app.services')
+.provider('pageProvider', function() {
+
+	var templates = {
+		'Default': {
```

Comment:
Can we change the name of this be less likely to collide with any other future resource names? While ideally I would like to use a Symbol, for better compatibility how about something just like `__defaults__`?